R/latex.R

```diff
@@ -18,6 +18,7 @@
 #' will fall back to \code{emulation = TRUE}. You can set the global option
 #' \code{options(tinytex.latexmk.emulation = FALSE)} to always avoid emulation
 #' (i.e., always use the executable \command{latexmk}).
+#'
```

Comment:
In the future, please refrain from touching non-essential lines in the code (i.e. making cosmetic changes). Thanks! :)